### DISCUSSION TITLE
안녕하세요 프론트엔드 개발 공부중인 직장인입니다.

### DISCUSSION THREAD

**Discussion by gusdas** [LifeIsAlone/public-home#24] [Q&A] [answered]
안녕하세요 프론트엔드 개발 공부중인 직장인입니다. 
공공 주택에 관심이 있는데 react로 작성되어있는거 보고 도움이 되고 싶어서 글 남깁니다. 
사용자 피드백 개선에 도움이 될 수 있을까요?

제 이메일 gustmd213@naver.com 입니다. 
꼭 같이 진행 해보고 싶습니다.
연락 기다리겠습니다!!

**Comment by cyberjam** [comment, marked as answer]
@gusdas 
안녕하세요! 프론트 겸 데이터 겸 벡엔드 겸사겸사 살고 있는 백수2입니다.  (제 동료 백수1도 계셔서..저는 2로)
이렇게 먼저 연락주셔서 정말 감사합니다! 

한동안 다른 프로젝트로 gitlab에서 놀고 있어서 github를 확인하지 못했네요 😫
현재는 다른 프로젝트 마무리하는 단계이고 다음달부터 다시 매입임대 주택시즌을 대비하여 열심히 코딩을 할 예정입니다.

늦게 확인 하셔서 답답하셨을것 같습니다. 오늘 중으로 직장인님 메일로 제 개인 메일 알려드리겠습니다. 관심 가져주셔서 감사합니다!

**Reply by cyberjam** [reply]
아! 제가 이메일 드리는걸 잊었군요. 그럼 팀원과 상의하여 남겨주신 카톡이나 디스코드로 연락하는 방향으로 잡겠습니다.
